I have now run the build in the command line outside of vscode and im still getting this error:

> (venv) D:\Pycharm projects\MY-PACKAGE-DEV\MY-PACKAGE>pyinstaller .\manage.spec --log-level CRITICAL
Traceback (most recent call last):
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.3568.0_x64__qbz5n2kfra8p0\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.3568.0_x64__qbz5n2kfra8p0\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "D:\Pycharm projects\MY-PACKAGE-DEV\MY-PACKAGE\venv\Scripts\pyinstaller.exe\__main__.py", line 7, in <module>
  File "D:\Pycharm projects\MY-PACKAGE-DEV\MY-PACKAGE\venv\lib\site-packages\PyInstaller\__main__.py", line 179, in run
    run_build(pyi_config, spec_file, **vars(args))
  File "D:\Pycharm projects\MY-PACKAGE-DEV\MY-PACKAGE\venv\lib\site-packages\PyInstaller\__main__.py", line 60, in run_build
    PyInstaller.building.build_main.main(pyi_config, spec_file, **kwargs)
  File "D:\Pycharm projects\MY-PACKAGE-DEV\MY-PACKAGE\venv\lib\site-packages\PyInstaller\building\build_main.py", line 963, in main
    build(specfile, distpath, workpath, clean_build)
  File "D:\Pycharm projects\MY-PACKAGE-DEV\MY-PACKAGE\venv\lib\site-packages\PyInstaller\building\build_main.py", line 885, in build
    exec(code, spec_namespace)
  File ".\manage.spec", line 7, in <module>
    a = Analysis(
  File "D:\Pycharm projects\MY-PACKAGE-DEV\MY-PACKAGE\venv\lib\site-packages\PyInstaller\building\build_main.py", line 410, in __init__
    self.__postinit__()
  File "D:\Pycharm projects\MY-PACKAGE-DEV\MY-PACKAGE\venv\lib\site-packages\PyInstaller\building\datastruct.py", line 173, in __postinit__
    self.assemble()
  File "D:\Pycharm projects\MY-PACKAGE-DEV\MY-PACKAGE\venv\lib\site-packages\PyInstaller\building\build_main.py", line 579, in assemble
    self.graph.process_post_graph_hooks(self)
  File "D:\Pycharm projects\MY-PACKAGE-DEV\MY-PACKAGE\venv\lib\site-packages\PyInstaller\depend\analysis.py", line 329, in process_post_graph_hooks
    module_hook.post_graph(analysis)
  File "D:\Pycharm projects\MY-PACKAGE-DEV\MY-PACKAGE\venv\lib\site-packages\PyInstaller\depend\imphook.py", line 447, in post_graph
    self._load_hook_module(keep_module_ref=True)
  File "D:\Pycharm projects\MY-PACKAGE-DEV\MY-PACKAGE\venv\lib\site-packages\PyInstaller\depend\imphook.py", line 387, in _load_hook_module
    self._hook_module = importlib_load_source(self.hook_module_name, self.hook_filename)
  File "D:\Pycharm projects\MY-PACKAGE-DEV\MY-PACKAGE\venv\lib\site-packages\PyInstaller\compat.py", line 609, in importlib_load_source
    return mod_loader.load_module()
  File "<frozen importlib._bootstrap_external>", line 529, in _check_name_wrapper
  File "<frozen importlib._bootstrap_external>", line 1029, in load_module
  File "<frozen importlib._bootstrap_external>", line 854, in load_module
  File "<frozen importlib._bootstrap>", line 274, in _load_module_shim
  File "<frozen importlib._bootstrap>", line 711, in _load
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "D:\Pycharm projects\MY-PACKAGE-DEV\MY-PACKAGE\venv\lib\site-packages\PyInstaller\hooks\hook-django.py", line 31, in <module>
    settings_py_imports = django.django_dottedstring_imports(root_dir)
  File "D:\Pycharm projects\MY-PACKAGE-DEV\MY-PACKAGE\venv\lib\site-packages\PyInstaller\utils\hooks\django.py", line 35, in django_dottedstring_imports
    ret = eval_script('django_import_finder.py', env=env)
  File "D:\Pycharm projects\MY-PACKAGE-DEV\MY-PACKAGE\venv\lib\site-packages\PyInstaller\utils\hooks\__init__.py", line 181, in eval_script
    return eval(txt)
  File "<string>", line 1
    2022-11-21 20:34:54,478::DEBUG::PyInstaller.utils.hooks::collect_submodules::650::Collecting submodules for django.contrib.contenttypes.templatetags
               ^
SyntaxError: invalid syntax

--- a/PyInstaller/hooks/rthooks/pyi_rth_django.py
+++ b/PyInstaller/hooks/rthooks/pyi_rth_django.py
@@ -11,53 +11,7 @@
 
 # This Django rthook was tested with Django 1.8.3.
 
-import django.core.management
 import django.utils.autoreload
-
-
-def _get_commands():
-    # Django groupss commands by app. This returns static dict() as it is for django 1.8 and the default project.
-    commands = {
-        'changepassword': 'django.contrib.auth',
-        'check': 'django.core',
-        'clearsessions': 'django.contrib.sessions',
-        'collectstatic': 'django.contrib.staticfiles',
-        'compilemessages': 'django.core',
-        'createcachetable': 'django.core',
-        'createsuperuser': 'django.contrib.auth',
-        'dbshell': 'django.core',
-        'diffsettings': 'django.core',
-        'dumpdata': 'django.core',
-        'findstatic': 'django.contrib.staticfiles',
-        'flush': 'django.core',
-        'inspectdb': 'django.core',
-        'loaddata': 'django.core',
-        'makemessages': 'django.core',
-        'makemigrations': 'django.core',
-        'migrate': 'django.core',
-        'runfcgi': 'django.core',
-        'runserver': 'django.core',
-        'shell': 'django.core',
-        'showmigrations': 'django.core',
-        'sql': 'django.core',
-        'sqlall': 'django.core',
-        'sqlclear': 'django.core',
-        'sqlcustom': 'django.core',
-        'sqldropindexes': 'django.core',
-        'sqlflush': 'django.core',
-        'sqlindexes': 'django.core',
-        'sqlmigrate': 'django.core',
-        'sqlsequencereset': 'django.core',
-        'squashmigrations': 'django.core',
-        'startapp': 'django.core',
-        'startproject': 'django.core',
-        'syncdb': 'django.core',
-        'test': 'django.core',
-        'testserver': 'django.core',
-        'validate': 'django.core'
-    }
-    return commands
-
 
 _old_restart_with_reloader = django.utils.autoreload.restart_with_reloader
 
@@ -70,9 +24,6 @@ def _restart_with_reloader(*args):
     finally:
         sys.argv.insert(0, a0)
 
-
-# Override get_commands() function otherwise the app will complain that there are no commands.
-django.core.management.get_commands = _get_commands
 # Override restart_with_reloader() function, otherwise the app might complain that some commands do not exist;
 # e.g., runserver.
 django.utils.autoreload.restart_with_reloader = _restart_with_reloader

